hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -122,6 +123,10 @@
   private static final String MERGE_DATA_VALIDATION_CHECK_ENABLED = "hoodie.merge.data.validation.enabled";
   private static final String DEFAULT_MERGE_DATA_VALIDATION_CHECK_ENABLED = "false";
 
+  // Enable multi writer support
+  private static final String HOODIE_TABLE_MULTIWRITER_ENABLED_PROP = "hoodie.table.multiwriter.enabled";
```

Comment:
repeating my earlier comment; lets just use the concurrency mode config from the other PR .

Reply:
Responded earlier